bigquery/indexer.py

```diff
@@ -54,6 +54,26 @@ def open_and_return_json(file_path):
     return jsonDict
 
 
+# Keep in sync with convert_to_index_name() in data-explorer repo.
+def convert_to_index_name(s):
+  """Converts a string to an Elasticsearch index name."""
+  # For Elasticsearch index name restrictions, see
+  # https://github.com/DataBiosphere/data-explorer-indexers/issues/5#issue-308168951
+  s = s.replace(' ', '_')
+  s = s.replace('"', '_')
+  s = s.replace('*', '_')
+  s = s.replace('\\', '_')
+  s = s.replace('<', '_')
+  s = s.replace('|', '_')
+  s = s.replace('\,', '_')
+  s = s.replace('>', '_')
+  s = s.replace('/', '_')
+  s = s.replace('?', '_')
+  s = s.lower()
+  print('Index name: %s' % s)
+  return s
```

Comment:
Use regexp to remove leading `_`?

Reply:
Done. (Added these to things that should be unit tested.)